js/platform-downloads.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+    var script = document.currentScript;
+    if (!script) { // polyfill for IE
+        var scripts = document.getElementsByTag('script');
+        script = scripts[scripts.length - 1]; // depends on synchronous load
+    }
+    var latest = script.dataset.latest,
+        platform = navigator.platform.toLowerCase();
```

Comment:
but you could use `navigator.userAgent` (I know that it's easy to fake it), but the majority doesn't. E.g. my Chrome sends this funny string to the world, 

```
navigator.userAgent
"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36"
```

(OT: the invention "feature detection" never made it to the browser world)